recipes/oats/create.sql

```diff
@@ -23,7 +23,7 @@ CREATE TEMP TABLE tmp (
     disability text,
     hhld_inc text,
     net_worth text,
-    location text
+    region text
```

Comment:
Why are we replacing location with region?

Reply:
@AmandaDoyle  It isn't replaced in the final output. I changed the name of what gets passed from python to postgres. Using python, I set a flag to indicate whether a zipcode is within the 31 county region (that's why I call it region). I decided to use postgres to separate the NYC from non-NYC zipcodes. This is because I was getting inconsistent results between using our city-zip lookup and doing a join with doitt zipcode boundaries. Since we use the doitt data for other zipcode level information, I wanted it consistent.

Reply:
See `location` assignment logic [here](https://github.com/MODA-NYC/db-recovery-data-partnership/blob/06f01c8dd4c4aace0ab5f6f2537ca61f9fcff560/recipes/oats/create.sql#L56-L66)